.github/taskfiles/build.yml

```
@@ -0,0 +1,47 @@
+  install:
+    desc: Install all dependencies using uv
+    deps: [uv]
+    cmds:
+      - |
+        # Check if .venv folder already exists
+        if [ -d ".venv" ]; then
+          printf "${BLUE}[INFO] Virtual environment already exists, skipping installation${RESET}\n"
+        else
+          # we need the virtual environment at least for the tests to work
+          # even if we don't install anything
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)